pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -176,7 +176,7 @@ func (c *customPluginMonitor) generateStatus(result cpmtypes.Result) *types.Stat
 
 					condition.Status = status
 					condition.Reason = result.Rule.Reason
-				} else if condition.Status == status && condition.Reason != result.Rule.Reason {
+				} else if condition.Status == status && (condition.Reason != result.Rule.Reason || (*c.config.PluginGlobalConfig.EnableMessageChangeBasedConditionUpdate && condition.Message != result.Message)) {
```

Comment:
The comment on line 180 needs update.

Can you also update the custom plugin readme to include this new option?

Reply:
@wangzhen127 Thanks for the tips. Will do.

Reply:
@wangzhen127 Done. PTAL.

Reply:
This line is way too long... can we break it into 2 lines at some point?